app/views/two_factor_mailer/backup_code.text.erb

```diff
@@ -0,0 +1,17 @@
+Hi <%= @user&.username %>,

```

Comment:
Does backup_code.html.erb replace this?

Reply:
They supplement each other - we send the email multipart so that we can send both content types - if someone can't read or doesn't want HTML email, they can use the plain text form.

Reply:
Ah.  The text version includes the code, so I thought it was a leftover.

Reply:
Is the text version intended to have the same wording as the HTML version, and direct the user to look up their backup code rather than including it in the email?

Reply:
Whoops, missed that.